tendrl/node_agent/gluster_integration/flows/import_cluster.py

```diff
@@ -17,26 +15,30 @@ def get_package_name(installation_source_type):
         return gluster
```

Comment:
I feel we can have a configuration defined say `gluster_integration_pip_install_url` under `[node_agent]` section of configuration and refer the same here.
Personally would prefer to do in a separate bug. Will raise an issue for the same.